### PR TITLE
We can set a max number of items that can be selected for bulk action

### PR DESCRIPTION
**Pull request by tdesvenain**

### CODE REVIEW

File: src/collective/contact/facetednav/browser/static/javascript.js [hunk 1]
```diff
@@ -25,7 +27,9 @@ contactfacetednav.init = function() {
                     contact.render();
                 });
             }
+            window.setTimeout(function() {
             contactfacetednav.contacts.render();
+            }, 500);
```
Comment (Gagaro) [inline src/collective/contact/facetednav/browser/static/javascript.js:32]: Why 500? Did you try 0 to see if the delayed call was enough?

File: src/collective/contact/facetednav/browser/static/javascript.js [hunk 2]
```diff
@@ -176,23 +181,41 @@ contactfacetednav.Contacts = Backbone.Collection.extend({
...
+            jQuery('.contacts-buttons input.global-selection').attr('disabled', true);
+            select_all_button.attr('disabled', false);
             select_all_button.attr('title', select_all_button.attr('data-unselect-all-msg'));
             select_all_button.attr('value', select_all_button.attr('data-unselect-all-msg'));
         }
         jQuery('#contacts-selection-num .num').text(selection_length);
+
+    },
+    resultTooLargeForSelectAll: function() {
+        return !!contactfacetednav.SELECT_ALL_MAX && this.length > contactfacetednav.SELECT_ALL_MAX
```
Comment (Gagaro) [inline src/collective/contact/facetednav/browser/static/javascript.js:218]: `;` 😱

File: src/collective/contact/facetednav/browser/static/javascript.js (hunk 1 above)
Comment (tdesvenain) [reply src/collective/contact/facetednav/browser/static/javascript.js:32]: fixed !

File: src/collective/contact/facetednav/browser/static/javascript.js (hunk 2 above)
Comment (tdesvenain) [reply src/collective/contact/facetednav/browser/static/javascript.js]: fixed

**Pull request merged**